files/fr/web/css/path()/index.html

```diff
@@ -0,0 +1,61 @@
+---
+title: path()
+slug: Web/CSS/path()
+translation_of: 'Web/CSS/path()'
+---
+<div>{{CSSRef}}</div>
+
+<p>La <a href="/fr/docs/Web/CSS/CSS_Functions">fonction</a> <a href="/fr/docs/Web/CSS">CSS</a> <code><strong>path()</strong></code> accepte comme paramètre une chaîne représentant un tracé SVG. Elle est utilisée dans les formes CSS et les animations de tracés CSS pour permettre de dessiner une forme.</p>
+
+<h2 id="syntax">Syntaxe</h2>
+
+<pre class="brush: css">path( [[&lt;'fill-rule'&gt;,]?&lt;string&gt;)</pre>
+
+<h3 id="parameters">Paramètres</h3>
+
+<dl>
+ <dt><code>&lt;'fill-rule'&gt;</code></dt>
+ <dd>La règle de remplissage de l'intérieur du tracé. Les valeurs possibles sont <i>nonzero</i> ou <i>evenodd</i>. La valeur par défaut est <i>nonzero</i>. Voir <a href="/fr/docs/Web/SVG/Attribute/fill-rule">fill-rule</a> pour plus de détails.</dd>
+ <dt>&lt;string&gt;</dt>
```

Comment:
to be enclosed between \<code> (even if en-US doesn't (yet))

Reply:
Honestly I hesitated ^^
Fixed in https://github.com/mdn/translated-content/pull/1747/commits/fb717b0df16bb78f0faa9ccc5d5377ceb56533bc

Reply:
Oops :) I was actually seeing the &amp;lt;string&amp;gt; in the second &lt;dt&gt; and did not spot the earlier code words.
Could you add another pair of &lt;code&gt;? (You can leave fb717b0 as is, it's fine :) )